vm/isolated/isolated.go

```diff
@@ -24,9 +25,11 @@ func init() {
 }
 
 type Config struct {
-	Targets      []string `json:"targets"`       // target machines: (hostname|ip)(:port)?
-	TargetDir    string   `json:"target_dir"`    // directory to copy/run on target
-	TargetReboot bool     `json:"target_reboot"` // reboot target on repair
+	Host         string   `json:"host"`           // host ip addr
+	Targets      []string `json:"targets"`        // target machines: (hostname|ip)(:port)?
+	TargetDir    string   `json:"target_dir"`     // directory to copy/run on target
+	TargetReboot bool     `json:"target_reboot"`  // reboot target on repair
+	USBDevNum    string   `json:"usb_device_num"` // usb device number
```

Comment:
you mean,
all variables, `TargetDir`, `TargetReboot`, `USBDevNum`, should be array?
or only `Targets` and `USBDevNum`?

Reply:
I mean only Targets and USBDevNum. Perhaps it makes sense to move then into a sub-struct so that address/port pairs are specified together.
syz-manager assumes that all test machines are equivalent, so I would assume that TargetDir and TargetReboot should be equal for all machines.